app/services/form_to_resource_service.rb

```diff
@@ -14,7 +14,9 @@ def convert(params, work, current_user)
       resource.publisher = params["publisher"] if params["publisher"].present?
       resource.publication_year = params["publication_year"] if params["publication_year"].present?
       resource.rights = PDCMetadata::Rights.find(params["rights_identifier"])
-      resource.keywords = (params["keywords"] || "").split(",")
+      resource.keywords = (params["keywords"] || "").split(",").map(&:strip)
+      resource.resource_type = params["resource_type"]
```

Comment:
This is in process_curator_controlled now, https://github.com/pulibrary/pdc_describe/blob/53e9f0d41bbb95cb6707737a20a7d75d5b6c6fe1/app/services/form_to_resource_service.rb#L42-L43

Reply:
Good catch, thanks!